### ISSUE TITLE
Store.clear() clears ALL stores

### ISSUE THREAD

**Issue by vothvovo** [sindresorhus/electron-store#243]
Is this intended? I feel like it is unintuitive.

**Comment by pierretusseau** [comment]
Yes as explained in the README : 

> ## `.reset(...keys)`
> Reset items to their default values, as defined by the defaults or schema option.
> 
> **Use .clear() to reset all items.**

**Comment by sindresorhus** [comment]
No, it clears the current store, as you can see here: https://github.com/sindresorhus/conf/blob/92320d7e2c9db2c56b05dd654c3a248a635598d2/source/index.ts#L285-L291

**Comment by JoanVicens** [comment]
I think the problem @vothvovo faced is that when you create 2 different stores they will overwrite each other. I assume it's because they share the same config.json file. I tried it myself on [this demo project](https://github.com/JoanVicens/electron-store-clear) to validate it.


```js
const Store = require('electron-store');

const store1 = new Store();
const store2 = new Store();


store1.set('unicorn', '🦄');
store2.set('unicorn', '🦄');

console.log(store1.get('unicorn'));
//=> '🦄'
console.log(store2.get('unicorn'));
//=> '🦄'

store1.clear();

console.log(store1.get('unicorn'));
//=> undefined
console.log(store2.get('unicorn'));
//=> undefined
```

**Comment by sindresorhus** [comment]
Both of your stores point at the same `config.json`, and `.clear()` clears that file, so they see the same data.

Give each store its own `name`, for example `new Store({name: 'a'})`, if you want separate files.
